ebr_connector/schema/build_results.py

```diff
@@ -38,6 +38,7 @@ class Test(InnerDoc):
         br_message: Any output from the test
         br_duration: Duration in milliseconds (float) of the test
         br_reportset: (Optional) Report set the test is a part of
+        br_context: (Optional) The context of the test
```

Comment:
Can this description be a bit more verbose? Something like "The runtime context of the test required to reproduce this execution" maybe?

Reply:
Done